packages/viewer/src/components/viewer-measurement-distance/viewer-measurement-distance.tsx

```diff
@@ -132,6 +136,13 @@ export class ViewerMeasurementDistance {
   @Prop({ mutable: true })
   public distance?: number;
 
+  /**
+   * Enables the display of axis reference lines between the start and end
+   * point.
+   */
+  @Prop()
+  public showAxisReferenceLines = false;
```

Comment:
Would there be any value in being able to specify the order the reference lines are displayed? Just testing around I've noticed that it can be a bit hard to follow the reference lines since they're always X <-> Y <-> Z from the second point. Wondering if something like Y <-> X <-> Z might be easier to follow (or if I'm just not used to this representation)